entrypoint.sh

```diff
@@ -1,26 +1,36 @@
 #!/bin/bash
 
+# config
+default_semvar_bump=${DEFAULT_BUMP:-minor}
```

Comment:
I like having a configurable default 👍 . Would you mind updating the README to document the optional env var? Probably in the `Usage` section.

Reply:
Yep - will do!

Reply:
I love the default as minor so I didn't put it in the main Usage example (didn't want to accidentally encourage people to set it manually), but added a section with options below.

In my own cloned repo I also added a DRY_RUN boolean flag that prevents pushing the tag (for soft releases, etc). Let me know if you want something like that.

Also am I wrong, does the SemVar library used here not support `v1.0.0` style SemVar tags, and only `1.0.0` style?

Reply:
The tool this project uses just spits out a semvar without the `v` prefix: https://github.com/fsaintjacques/semver-tool